porting-quick.md

```diff
@@ -59,18 +59,17 @@ The ```serial_s``` is for referencing memory-mapped Serial registers and passing
 
 ### US TICKER
 
-The microsecond ticker is a system resource that is used for many API and other timing utilities. It needs a one microsecond resolution and should be implemented using a free-running hardware counter or timer with match register.
-The required implementation definition is in ```mbed-os\hal\us_ticker_api.h```.
+The microsecond ticker is a system resource that many APIs and other timing utilities use. It needs a one microsecond resolution and should be implemented using a free-running hardware counter or timer with match register. The required implementation definition is in ```mbed-os\hal\us_ticker_api.h```.
 
 At this point, we should be able to compile a handful of  tests: 
 
 ``mbed test -m BOARD_NAME --compile -t <toolchain>``
 
-To execute the tests you'll need to already support [mbed-ls](https://github.com/armmbed/mbed-ls).
+To execute the tests, you'll need to already support [mbed-ls](https://github.com/armmbed/mbed-ls).
 
 ### All the others
 
-At this point, the HAL structure should be familiar as a programming model. There are many more APIs to implement, which are enabled by adding a ```device_has``` attribute to the MCU_NAME and then providing the implementation. Other HAL apis include but are not limited to:
+At this point, the HAL structure should be familiar as a programming model. There are many more APIs to implement, which you enable by adding a ```device_has``` attribute to the MCU_NAME and then providing the implementation. Other HAL APIs include but are not limited to:
 
 ```
 device_has       |   api
```

Comment:
Do we want to include the "Coming soon" message at the end of the page? Is that info still coming soon?